rocAL/rocAL_pybind/examples/rocAL_api_video_pipeline.py

```diff
@@ -114,36 +125,33 @@ def main():
     display = args.display
     num_threads = args.num_threads
     random_seed = args.seed
-    tensor_format = types.NHWC if args.NHWC else types.NCHW
+    tensor_format = types.NFHWC if args.NHWC else types.NFCHW
     tensor_dtype = types.FLOAT16 if args.fp16 else types.FLOAT
     # Create Pipeline instance
-    pipe = Pipeline(batch_size=batch_size, num_threads=num_threads,device_id=args.local_rank, seed=random_seed, rocal_cpu=_rocal_cpu,
-                    mean=[0.485 * 255, 0.456 * 255, 0.406 * 255], std=[0.229 * 255, 0.224 * 255, 0.225 * 255], tensor_layout=tensor_format, tensor_dtype=tensor_dtype)
```

Comment:
Why is the mean and std dev removed from here ? Where is it added back ?

Reply:
I removed the mean and std here but haven't added it to the video pipeline since it produces normalized images when dumping the images